<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkMorphologicalOperationsWidgetControls</class>
 <widget class="QWidget" name="QmitkMorphologicalOperationsWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QmitkDataSelectionWidget" name="dataSelectionWidget" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Structuring Element</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonMorphoBall">
        <property name="text">
         <string>Ball</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonMorphoCross">
        <property name="text">
         <string>Cross</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="planeSelectionComboBox">
        <item>
         <property name="text">
          <string>3D Operation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2D Operation - Axial</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2D Operation - Sagital</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2D Operation - Coronal</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Radius</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderMorphFactor">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMorphFactor">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QToolButton" name="btnDilatation">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Dilation</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/MorphologicalOperationsWidget.qrc">
         <normaloff>:/SegmentationUtilities/MorphologicalOperations/Dilate_48x48.png</normaloff>:/SegmentationUtilities/MorphologicalOperations/Dilate_48x48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="btnErosion">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Erosion</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/MorphologicalOperationsWidget.qrc">
         <normaloff>:/SegmentationUtilities/MorphologicalOperations/Erode_48x48.png</normaloff>:/SegmentationUtilities/MorphologicalOperations/Erode_48x48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="btnClosing">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Closing</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/MorphologicalOperationsWidget.qrc">
         <normaloff>:/SegmentationUtilities/MorphologicalOperations/Closing_48x48.png</normaloff>:/SegmentationUtilities/MorphologicalOperations/Closing_48x48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="btnOpening">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Opening</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/MorphologicalOperationsWidget.qrc">
         <normaloff>:/SegmentationUtilities/MorphologicalOperations/Opening_48x48.png</normaloff>:/SegmentationUtilities/MorphologicalOperations/Opening_48x48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="btnFillHoles">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Globally fills holes in segmentation (structuring element and radius not required)</string>
       </property>
       <property name="text">
        <string>Fill Holes</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/MorphologicalOperationsWidget.qrc">
         <normaloff>:/SegmentationUtilities/MorphologicalOperations/FillHoles_48x48.png</normaloff>:/SegmentationUtilities/MorphologicalOperations/FillHoles_48x48.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>518</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataSelectionWidget</class>
   <extends>QWidget</extends>
   <header>internal/Common/QmitkDataSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../resources/MorphologicalOperationsWidget.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sliderMorphFactor</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxMorphFactor</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>766</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxMorphFactor</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sliderMorphFactor</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>784</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
